Assets/js/astro.js

```diff
@@ -1,9 +1,17 @@
 //JavaScript file for the Astronomy API
+const authString = btoa(`applicationId:applicationSecret`);
+
+import {lat,lon} from './geolocation.js';
```

Comment:
Slavic and Rene told us we didn't need to import, we could just call the code lat and lon and it should be fine. Worked for me in my weather.js file. As long geolocation.js is above the other script requests from js in modal.html